clang/lib/Parse/ParseOpenMP.cpp

```diff
@@ -2997,7 +2997,7 @@ bool Parser::ParseOpenMPSimpleVarList(
                 StopBeforeMatch);
     } else if (ParseUnqualifiedId(SS, /*ObjectType=*/nullptr,
                                   /*ObjectHadErrors=*/false, false, false,
-                                  false, false, nullptr, Name)) {
+                                  false, false, false, nullptr, Name)) {
```

Comment:
This is pretty gnarly - can we add comments (like below)?
I think a NFC patch that would turn that into flags or a struct would be nice